src/models/User/DateOfBirth.ts

```diff
@@ -24,4 +25,11 @@ export default class DateOfBirth {
   get(): Date {
     return this.dob;
   }
+
+  getFormattedDate(): string {
```

Comment:
Suggestion: This should be a simpler way of achieving the same goal.

```suggestion
  getFormatted(): string {
    return this.dob.toISOString().split('T')[0];
  }
```